src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -878,13 +878,25 @@ public static bool TryGetEntityMetadata(CallNode callNode, TexlBinding binding,
 
         // Fetch the description associated with the specified parameter name (which must be the INVARIANT name)
         // If the param has no description, this will return false.
-        public virtual bool TryGetParamDescription(string paramName, out string paramDescription)
+        public virtual bool TryGetParamDescription(string paramName, out string paramDescription, string locale = null)

```

Comment:
>locale [](http://example.com/codeflow?start=105&length=6)

We should test these 4 cases: 

- null 
- empty    // CultureInfo.InvariantCulture
- bogus // invalid value 
- En-US // valid, same as UI culture
- Fr-fr // valid, different than UI culture

Last point is interesting because of code like this:
[https://github.com/microsoft/Power-Fx/blob/59aa6c1a63e3b92f73f2c0d40825a90bdd5d580b/src/libraries/Microsoft.PowerFx.Core/Localization/StringResources.cs#L94C11-L99C1](https://github.com/microsoft/Power-Fx/blob/59aa6c1a63e3b92f73f2c0d40825a90bdd5d580b/src/libraries/Microsoft.PowerFx.Core/Localization/StringResources.cs#L94C11-L99C1)